docs/examples/config-template/sealedsecret.yaml

```diff
@@ -22,15 +22,15 @@ spec:
                                "port": "22",
                                "env": "SOME_ENV",
                                "user": "myuser",
-                               "password": "{{ index . "server1" }}",
+                               "password": {{ toJson .server1 }},
```

Comment:
I snuck in an unrelated change here (`index . "server1"` to `.server1`). Happy to back it out if folks prefer. I prefer it because it avoids parens:

```
"password": "{{ toJson (index . "server1") }}",
```

(but, total go noob, not sure if there are other considerations)